cg/apps/scoutapi.py

```diff
@@ -36,6 +37,6 @@ def upload(self, data: dict, threshold: int=5, force: bool=False):
             LOG.debug("load new Scout case")
             load_scout(self, config_data)
 
-    def export_panels(self, panels: List[str]):
+    def export_panels(self, panels: List[str], versions: None):
```

Comment:
Please refrain from changing function signatures like this. I presume you wanted to add a default to `versions` and not demand that `versions` is of type `None` ;)

Reply:
Fixed in bdc72a2e4eb7d6f7 :)